helpers-assets/templates/translation-discussion-comments.php

```diff
@@ -7,53 +7,87 @@
 	<?php if ( $number = count( $comments ) ) : ?>
 		<h6><?php printf( _n( '%s Comment', '%s Comments', $number ), number_format_i18n( $number ) ); ?>
 		<?php if ( $locale_slug ) : ?>
-			(<?php echo esc_html( $locale_slug )?>)
+			(<?php echo esc_html( $locale_slug ); ?>)
 			<?php
-				$countLocaleComments = 0;
-				foreach ( $comments as $comment ) {
-					$comment_locale = get_comment_meta( $comment->comment_ID, 'locale', true );
-					if ( $locale_slug == $comment_locale ) {
-						$countLocaleComments++ ;
-					}
-				}	
-			?>			
+			$countLocaleComments = 0;
+			foreach ( $comments as $comment ) {
+				$comment_locale = get_comment_meta( $comment->comment_ID, 'locale', true );
+				if ( $locale_slug == $comment_locale ) {
+					$countLocaleComments++;
+				}
+			}
+			?>
+						
 			<span class="comments-selector">
-				<a href="#" class="active-link" data-selector="all">Show all (<?php echo esc_html( $number ); ?>)</a> | <a href="#" data-selector="<?php echo esc_attr( $locale_slug );?>"><?php echo esc_html( $locale_slug );?> only (<?php echo esc_html( $countLocaleComments ); ?>)</a>
+				<a href="#" class="active-link" data-selector="all">Show all (<?php echo esc_html( $number ); ?>)</a> | <a href="#" data-selector="<?php echo esc_attr( $locale_slug ); ?>"><?php echo esc_html( $locale_slug ); ?> only (<?php echo esc_html( $countLocaleComments ); ?>)</a>
 			</span>
 		<?php endif; ?>
 		</h6>
 	<?php endif; ?>
 	<ul class="discussion-list">
 		<?php
-		wp_list_comments( array(
-			'style'       => 'ul',
-			'type'       => 'comment',
-			'callback' => 'gth_discussion_callback',
-			'translation_id' => $translation_id,
-			'locale_slug' => $locale_slug,
-			'original_permalink' => $original_permalink,
-		), $comments );
+		wp_list_comments(
+			array(
+				'style'              => 'ul',
+				'type'               => 'comment',
+				'callback'           => 'gth_discussion_callback',
+				'translation_id'     => $translation_id,
+				'locale_slug'        => $locale_slug,
+				'original_permalink' => $original_permalink,
+			),
+			$comments
+		);
 		?>
 	</ul><!-- .discussion-list -->
 	<?php
+	add_action(
+		'comment_form_logged_in_after',
+		function () use ( $locale_slug ) {
+			$english_name_for_locale = '';
+			$language_question       = '';
+
+			if ( $locale_slug ) {
+				$english_name_for_locale = GP_Locales::by_slug( $locale_slug )->english_name;
+				$language_question       = '<option value="question">Question about translating to ' . esc_html( $english_name_for_locale ) . '</option>';
```

Comment:
For more defensive programming, please do check if the locale could be found, something like
```suggestion
				$gp_locale = GP_Locales::by_slug( $locale_slug );
				if ( $gp_locale ) {
					$language_question       = '<option value="question">Question about translating to ' . esc_html( $gp_locale->english_name ) . '</option>';
				}
```